app/views/feedback/steps/completed.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @hide_page_helpful_question = true %>
+
+<section>
+  <h1 class="heading-l heading--box-green feedback-thankyou">Thank you for your feedback</h1>
```

Comment:
If this is easy @StackedPancakez, can we change this heading background to be blue rather than green, to match the mailing list completion page - we are trying to use green just for CTAs

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/dc4a35b9-0ebb-4b12-9693-5edccaaa9e82)